lib/stale.js

```diff
@@ -108,13 +108,15 @@ module.exports = class Stale {
     const {owner, repo} = this.config
     const perform = this.getConfigValue(type, 'perform')
     const staleLabel = this.getConfigValue(type, 'staleLabel')
-    const markComment = this.getConfigValue(type, 'markComment')
+    var markComment = this.getConfigValue(type, 'markComment')
```

Comment:
Let's move this below the constants since it's a different type of variable.

Reply:
Shouldn't we place all `this.getConfigValue` statements together first?

Reply:
It doesn't affect the value, so no?

Reply:
Done, thanks!